src/asm/charmap.cpp

```diff
@@ -16,10 +17,11 @@
 // Essentially a tree, where each nodes stores a single character's worth of info:
 // whether there exists a mapping that ends at the current character,
 struct CharmapNode {
-	bool isTerminal; // Whether there exists a mapping that ends here
-	uint8_t value;   // If the above is true, its corresponding value
+	std::vector<int32_t> value;   // The mapped value, if there exists a mapping that ends here
```

Comment:
I miss C's flexible array member... u_u

Reply:
Yeah, I wish this had been a two dozenth realloc'ed-array implementation too :P